tests/test_wave_01.py

```diff
@@ -139,14 +140,15 @@ def test_moves_movie_from_watchlist_to_watched():
     updated_data = watch_movie(janes_data, movie_to_watch["title"])
 
     # Assert
-    assert len(updated_data["watchlist"]) is 1
-    assert len(updated_data["watched"]) is 2
+    assert len(updated_data["watchlist"]) == 1
+    assert len(updated_data["watched"]) == 2
+    assert updated_data["watched"] == [FANTASY_2, HORROR_1]
```

Comment:
If we couldn't guarantee the order of the watched list, we would want our assert to be more flexible. We could use :
```suggestion
    assert FANTASY_2 in updated_data["watched"]
    assert HORROR_1 in updated_data["watched"]
```